awesome_book.html

```diff
@@ -4,25 +4,43 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="assets/styles/style.css">
+  <link rel="stylesheet" 
+        href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" 
+        integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" 
+        crossorigin="anonymous">
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" 
+          integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" 
+          crossorigin="anonymous">
+  </script>
+  <script src="https://kit.fontawesome.com/9f43e9dbcb.js" crossorigin="anonymous"></script> 
+
   <title>Awesome Book</title>
 </head>
 <body>
-  <div>
-  <h1>Awesome Books</h1>
-  <div id="list">
-  
-  </div>
-  <div id="message">
-
+  <div class="container-fluid my-5 d-flex align-items-center justify-content-center">
+    <div class="row mx-auto">
+      <div class="col-md-12">
+        <h1 class="mb-4 text-center"> All awesome books </h1>
+        <ul id="list" class="list-unstyled rounded"></ul>
+        <div id="message"></div>
+        <div class="d-flex align-items-center justify-content-center flex-column">
+          <h1 class="h2 mt-3">Add a new book</h1>
+          <form id="form">
+           <div class="form-group my-3">
+            <input type="text" name="title" id="title" class="form-control" placeholder="Title">
+           </div>
+            <div class="form-group my-3">
+              <input type="text" name="author" id="author" class="form-control" placeholder="Author">
+            </div>
+            <div class="form-group my-3 d-flex align-items-center justify-content-end">
+              <input type="submit" value="Add" id="add" class="button border-black">
+            </div>
+          </form>
+        </div>
+      </div>
+    </div>
   </div>
-  <form id="form">
-    <input type="text" name="title" id="title">
-    <br><br>
-    <input type="text" name="author" id="author">
-    <br><br>
-    <input type="submit" value="Add" id="add">
-  </form>
-</div>
   <script src="js/book_list.js"></script>
```

Comment:
- [ ] Please use proper semantic HTML tags here, in order to keep with the best practices. For example, use `<header>` for the page title and `<main>` for the body.